core/node/state_keeper/src/mempool_actor.rs

```diff
@@ -81,7 +83,8 @@ impl MempoolFetcher {
                 break;
             }
             let latency = KEEPER_METRICS.mempool_sync.start();
-            let mut storage = self.pool.connection_tagged("state_keeper").await?;
+            let mut connection = self.pool.connection_tagged("state_keeper").await?;
+            let mut storage = connection.start_transaction().await?;
```

Comment:
nit: please change name of this var to `transaction` so it will be more clear that executed queries are transactional in code

Reply:
I will use `storage_transaction`, it feels like the word `transaction` will collide too much in this context with actual L2 transactions